### ISSUE TITLE
Cannot connect to Pod via Exec

### ISSUE THREAD

**Issue by vmpowercli** [lensapp/lens#2753]
**Describe the bug**
Cannot connect to pod via exec
Unable to connect to the server: dial tcp 127.0.0.1:63288: socket: The requested service provider could not be loaded or initialized.

**To Reproduce**
1. Go to  Lens
2. Click on pod and then click on exec shell
3. Scroll down to terminal screen for error
4. See "Unable to connect to the server: dial tcp 127.0.0.1:63288: socket: The requested service provider could not be loaded or initialized."

**Expected behavior**
I was working fine on version 3 Lens.. after upgrading to latest version I cannot exec in to pods

**Screenshots**
![image](https://user-images.githubusercontent.com/21179398/117839466-3ebf1900-b249-11eb-96f4-a66c373df596.png)

**Environment (please complete the following information):**
- Lens Version: 4.2.4
- OS: Windows
- Installation method from .exe

Additional Steps
I even added the proxy On the cluster preference but still no luck

**Comment by vmpowercli** [comment]
kubectl.exe get all
Unable to connect to the server: dial tcp 127.0.0.1:50276: socket: The requested service provider could not be loaded or initialized.
Unable to connect to the server: dial tcp 127.0.0.1:50276: socket: The requested service provider could not be loaded or initialized.
Unable to connect to the server: dial tcp 127.0.0.1:50276: socket: The requested service provider could not be loaded or initialized.
Unable to connect to the server: dial tcp 127.0.0.1:50276: socket: The requested service provider could not be loaded or initialized.
Unable to connect to the server: dial tcp 127.0.0.1:50276: socket: The requested service provider could not be loaded or initialized.
Unable to connect to the server: dial tcp 127.0.0.1:50276: socket: The requested service provider could not be loaded or initialized.
Unable to connect to the server: dial tcp 127.0.0.1:50276: socket: The requested service provider could not be loaded or initialized.
Unable to connect to the server: dial tcp 127.0.0.1:50276: socket: The requested service provider could not be loaded or initialized.
Unable to connect to the server: dial tcp 127.0.0.1:50276: socket: The requested service provider could not be loaded or initialized.
Unable to connect to the server: dial tcp 127.0.0.1:50276: socket: The requested service provider could not be loaded or initialized.

**Comment by vmpowercli** [comment]
Any help?

**Comment by jim-docker** [comment]
Sorry, I lost sight of this. It looks like it is trying to exec `powershell` in the container? Is it a windows container running on the pod? This is what I see when trying to do a similar thing:
```
kubectl exec -i -t -n default my-nginx-666d688fd-pn2lp -c my-nginx "--" sh -c "clear; (bash || ash || sh)"
```
What do you have set for the shell path (on the Lens preferences page)?

**Comment by mjseid** [comment]
@vmpowercli  were you able to resolve this?  seeing similar

**Comment by vmpowercli** [comment]
yes, the issues still exists and other person also reported the same issue 
https://github.com/lensapp/lens/issues/3756

Thank you
